README.md

```diff
@@ -1,59 +1,134 @@
 # Loop Exercises
 
-1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
-For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
-For each iteration of the loop, it will multiply the number by 9 and log the result (e.g. "2 * 9 = 18").
-
-_Bonus_ think of another way to solve it. 
-  <details>
-    <summary>
-      Hint
-    </summary>
-    Find the final number and increment the loop by 9.
-  </details>
-  
-4. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions. For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
-
-
-5. Modify your program to log "FizzBuzz", for numbers that are divisible by both 3 and 5 (still log "Fizz" or "Buzz" for numbers divisible by only one of those).
-
-
-Bonus: 
-
-1. Write a program that would log the lyrics of the song 99 Bottles of Beer. This is the first verse of the song:
-
-99 bottles of beer on the wall,
-
-99 bottles of beer.
-
-Take one down, pass it around,
-
-98 bottles of beer on the wall.
-
-This verse is repeated, each time with one less bottle, until the number of bottles is 0. When the number of bottles is 2, the verse is:
-
-2 bottles of beer on the wall,
-
-2 bottles of beer.
-
-Take one down, pass it around,
-
-1 bottle of beer on the wall.
-
-In the last line, the word bottles (plural), is  replaced with bottle (singular)
-
-When the number of bottles is 1, the verse is:
-
-1 bottle of beer on the wall,
-
-1 bottle of beer.
-
-Take one down, pass it around,
-
-No more bottle of beer on the wall.
-
+//1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
+
+//while loop:
+
+let num = 13
+
+while(num >= 1){
+  console.log("the number is " + num)
+  num--
+}
+
+//for loop:
+ let i = 13
+ for(i = 13; i >= 1; i -= 1) {
+  console.log(i)
+}
+
+
+//2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
+//For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+
+//while loop:
+
+let num = 0;
+
+while(num <= 33 ){
+  if(num % 2 === 0){
+  console.log(num + " is even")
+  } if (num % 2 !== 0) {
```

Comment:
use 'else' instead of if here. if (condition) do this, else do that. If you need to check another condition use else if.